index.html

```diff
@@ -93,9 +93,11 @@ <h3>Move Map</h3>
 <div id="config-modal">
     <div class="content">
         <h1>Map configuration</h1>
-        <p><b>Warning:</b> Deleting tiles or icons here or reducing the number of floors will delete any points of those types in your map!</p>
+        <p><b><font color="red">Warning:</font></b> Deleting tiles/icons here, or reducing the number of floors or tile side length, will <b>delete</b> any points of those types in your map!</p>
```

Comment:
pls use `<span class="warningbtn">` instead (feel free to rename just "warning" or w/e)

Reply:
Not sure where/how you want this used

Reply:
I'd modify the CSS to look like:
```
button.warning {
  float: right;
}
.warning {
  color: var(--color-err);
}
```
which will change the existing `warningbtn` class to `warning` and just apply the color (unless it's a `button`)
and then instead of `<font>` you just use the span tag